console/apps/webapp/public/config.js

```
@@ -18,26 +18,27 @@
+    scopes: ['openid', 'profile', 'email'],
+    platform: 'AsgardeoV2',
+    tokenValidation: {
+      idToken: {
+        // Disable for Thunder / local dev with non-standard issuers or self-signed certs
+        validate: false,
+        clockTolerance: 300,
+      },
+    },
+    storage: 'localStorage',
   },
   disableAuth: 'true' === 'true',
```

Biome (2.4.9)

[error] Comparing to itself is potentially pointless.

(lint/suspicious/noSelfCompare)